core/templates/components/save-pending-changes/save-pending-changes-modal.component.spec.ts

```diff
@@ -46,14 +46,11 @@ describe('Story Editor Save Modal Component', () => {
   }));
 
   beforeEach(() => {
-    fixture = TestBed.createComponent(StorySavePendingChangesModalComponent);
+    fixture = TestBed.createComponent(SavePendingChangesModalComponent);
     component = fixture.componentInstance;
-    ngbActiveModal = TestBed.inject(NgbActiveModal);
```

Comment:
Can we delete the mock modal defined above now that it is no longer required?

Reply:
Done